backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/AddUserProfilePropertyCommand.java

```diff
@@ -28,6 +31,8 @@ public AddUserProfilePropertyCommand(T parameters, CommandContext cmdContext) {
     @Override
     protected boolean validate() {
         UserProfileProperty newProp = getParameters().getUserProfileProperty();
+        addCustomValue(PROFILE_USER, newProp.getUserId().toString());
```

Comment:
Since `user ID` is not displayed as part of webadmin general info, it's much less convenient for admins to figure out which user is referred by this message. Note that `user ID` is not used on most of audit log messages while other entities does appear - since this info is exposed on webadmin.

I prefer to avoid that part of adding the target user till we find a way to solve it.

Or, if you insists :), just put that within parenthesis as an extra info for users that do want to query the DB for that.
Maybe, you can also add `property id` within those parenthesis as well in that case.

Reply:
> Bottom line, logging from the `commands` level is very confusing and inconsistent. i wonder if there is a way to generate those logs on a higher level for the vm portal and for reset settings at least.

@rszwajko what about moving to a upper level for logging a real user actions like "updating a property" or "resetting a property"? This has much more value than just logging each command...